openapi-cli/src/test/java/io/ballerina/openapi/generators/schema/PrimitiveDataTypeTests.java

```diff
@@ -82,7 +82,7 @@ public void generateMissingDatatype() throws IOException, BallerinaOpenApiExcept
     @Test(description = "When the component schema has primitive data type instead of object schema")
     public void generateSchemaForPrimitiveData() throws IOException, BallerinaOpenApiException {
         Path definitionPath = RES_DIR.resolve("swagger/schema_with_primitive.yaml");
-        OpenAPI openAPI = codeGenerator.normalizeOpenAPI(definitionPath);
+        OpenAPI openAPI = codeGenerator.normalizeOpenAPI(definitionPath, true);
```

Comment:
What does this second parameter `true` does?

Reply:
if `true` is given, new operationIds will not be generated for missing operationIds.